lib/jellyfish/room_service.ex

```diff
@@ -17,7 +16,7 @@ defmodule Jellyfish.RoomService do
   @spec find_room(Room.id()) :: {:ok, pid()} | {:error, :room_not_found}
   def find_room(room_id) do
     case Registry.lookup(Jellyfish.RoomRegistry, room_id) do
-      [{room_pid, nil}] ->
+      [{room_pid, _webhook_url}] ->
```

Comment:
I would change the `_webhook_url` to a different name as we never assign webhook to registry id.